html/changelogs/Mandurrrh-floortilefix.yml

```diff
@@ -0,0 +1,10 @@
+
+# Your name.  
+author: Mandurrrh
+
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+
+changes: 
+  -bugfix: "Name of light floor tiles got changed this fixes their names to work with all colors"
```

Comment:
Ya I guess I keep forgoing grammar with changelogs just quick statements. And I wasn't sure since there was no issue actually posted if it counted so I wanted to include it just in case. Want me to remove it?